### PR TITLE
Fix checkstyle

### PR DESCRIPTION
**Pull request by Kafcis**
I went through the code to fix the checkstyle issues

### CODE REVIEW

File: src/main/java/athena/Storage.java
Comment (alstontham) [inline src/main/java/athena/Storage.java]: would checkstyle flag this extra space?

File: src/main/java/athena/Storage.java
Comment (alstontham) [inline src/main/java/athena/Storage.java]: would checkstyle flag this extra space?

**Review by alstontham — COMMENTED** [review]
Looks good! Just a few extra empty lines that the checkstyle could possibly flag.

**Review by daniellimws — APPROVED** [review]
Since checkstyle approves I think this should be good. 

But we should only merge this after the other PRs are merged because some of the things here might have not been approved in the other PRs.

**Comment by daniellimws** [comment]
Er this PR has error with checkstyle test after you merged master into it. We shouldn't have merged.

**Pull request merged**
